src/calculateRentalCost.js

```diff
@@ -4,7 +4,21 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_PER_DAY = 40;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+
+  let price = days * PRICE_PER_DAY;
```

Comment:
Consider using `const` for the `price` variable since it's not being reassigned, but rather its value is being updated conditionally. This can help prevent accidental reassignment and make the code more robust.